src/t8_forest/t8_forest_private.cxx

```diff
@@ -57,6 +57,14 @@ t8_forest_get_tree_leaf_element_array_mutable (const t8_forest_t forest, t8_loci
   return (t8_element_array_t *) t8_forest_get_tree_leaf_element_array (forest, ltreeid);
 }
 
+/* TODO: does the search fail when element_level is smaller then levels in the array?
```

Comment:
Have you opened issues for these Todos? That way they do not get lost that easily.